content/20.treatments.md

```diff
@@ -226,6 +226,16 @@ In principle, drugs that reduce the expression of these proteins or sterically h
 
 A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
 
+Angiotensin-converting enzyme inhibitors (ACEis) and angiotensin II receptor blockers (ARBs) are among the most commonly prescribed medications, prescribed well over 100,000,000 times annually in the United States, for example.
+Data from some animal models suggest several but not all ACEis and several ARBs increase ACE2 expression in cells of some organs.
+Clinical studies have not established that ACE2 expression is increased in humans treated with these medications.
+While randomized clinical trials are ongoing, a variety of observational studies have analyzed the relationship between exposure to ACEis or ARBs and outcomes in patients with COVID-19.
+Mehra and colleagues reported a 0.33 odds of death in hospitalized patients with COVID-19 who were treated with an ACEi at time of admission.[@doi:10.1056/NEJMoa2007621]
```

Comment:
Is ACEi an extremely frequent shortening? As someone who doesn't work with these drugs every day, I feel like ACE inhibitor/s is more readable to me, but if this is very widely used then I won't object :)

Reply:
It's common, but if our readership won't find it familiar, I propose we use the expanded form.